src/compiler/checker.ts

```diff
@@ -3265,9 +3265,9 @@ namespace ts {
                 const outerTypeParameters = getOuterTypeParametersOfClassOrInterface(symbol);
                 const localTypeParameters = getLocalTypeParametersOfClassOrInterfaceOrTypeAlias(symbol);
                 // A class or interface is generic if it has type parameters or a "this" type. We always give classes a "this" type
-                // because it is not feasible to analyze all members to determine if the "this" type escapes the class (in particular,
+                // because it is not feasible to analyse all members to determine if the "this" type escapes the class (in particular,

```

Comment:
I normally would use `z`, but there was more instances of `s`, so I favored it against my own preference. I'll gladly switch :-)